spec/classes/profile/postfix_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+require 'spec_helper'
+
+describe 'nebula::profile::postfix' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+
+      it { is_expected.to compile }
```

Comment:
I would add an expectation to contain the postfix package and not contain exim.

Reply:
Again,  going to note this down and pass until I gain more experience.